cyclonedx/model/component.py

```diff
@@ -1148,7 +1152,12 @@ def get_vulnerabilities(self) -> "SortedSet[Vulnerability]":
         Returns:
              Set of `Vulnerability`
         """
-        return self.__vulnerabilites
+        warnings.warn(
```

Comment:
why deprecate it?  why not drop it entirely? 
its a breaking-changes-version anyway?

Reply:
Was draft code - agree entirely. It will be removed.